cpp/src/core/Transpose_Test/TransposeTest.cc

```diff
@@ -0,0 +1,34 @@
+// This file contains a simple tranpose function and a test to make sure that
+// the transpose worked
+
+#include "../../../../Eigen/Eigen/Dense"
```

Comment:
The Eigen directory now is included in the framework. So no need to use your own path. And also note that '../' is not encouraged in our code style.